src/app/components/site/Homepage.tsx

```diff
@@ -71,6 +72,70 @@ export const Homepage = () => {
           </section>
         )}
 
+        {!user?.loggedIn && (
+          <section id="resources" className="resources-section">
+            <Container className="pt-4 pb-5">
+              <div className="pt-5">
+                <h2 className="homepage-sub-title text-left mb-4">Resources</h2>
+              </div>
+              <Row>
+                <div className="columns-container">
+                  <Col className="custom-col-60">
+                    <Resources>
```

Comment:
Sorry no, I meant it more fleeting than just a wrapper. The whole content of this card could be part of this `Resources` component with the props powering how it works. There's a lot of structural repetition here that can be saved by enhancing what you've just created.